lms/djangoapps/grades/grade_utils.py

```diff
@@ -15,8 +15,9 @@
 from courseware.models import StudentModule
 from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
 from student.models import CourseEnrollment
-from super_csv import ChecksumMixin, CSVProcessor, DeferrableMixin
+from super_csv.csv_processor import ChecksumMixin, CSVProcessor, DeferrableMixin
```

Comment:
Hi there.  Folks from the BOM squad now automatically get notified when changes happen in certain parts of our core platform, per [CODEOWNERS](https://github.com/edx/edx-platform/blob/master/.github/CODEOWNERS).

May I kindly request that we consider keeping the core Grades layer as lean as possible so that it only owns the business logic for computing, storing, and retrieving grades.  This is the intention of the grades django app, as now described in its [README](https://github.com/edx/edx-platform/blob/master/lms/djangoapps/grades/README.rst).

To that end, features that export and format Grades in CSVs should be implemented in a separate layer/app.